frontend/ui/data/koptoptions.lua

```diff
@@ -30,15 +46,35 @@ local KoptOptions = {
             {
                 name = "trim_page",
                 name_text = S.PAGE_CROP,
-                toggle = {S.MANUAL, S.AUTO, S.SEMIAUTO, S.NONE},
+                -- manual=0, auto=1, semi-auto=2, none=3
+                -- ordered from least to max cropping done or possible
+                toggle = {S.NONE, S.AUTO, S.SEMIAUTO, S.MANUAL},
                 alternate = false,
-                values = {0, 1, 2, 3},
+                values = {3, 1, 2, 0},
                 default_value = DKOPTREADER_CONFIG_TRIM_PAGE,
                 enabled_func = Device.isTouchDevice,
                 event = "PageCrop",
-                args = {"manual", "auto", "semi-auto", "none"},
+                args = {"none", "auto", "semi-auto", "manual"},
                 name_text_hold_callback = optionsutil.showValues,
-            }
+                help_text = _([[Allows cutting out blank page margins in the original document.
```

Comment:
I'd keep using crop here, it should be common enough (i.e., cropping or croppping out instead of cutting) ;).

Reply:
Initially went with "crop out", but Google tells me it's only used in "(of rock) appear or be exposed at the surface of the earth." :)
Thought cutting out might be an alternative explanation for people not used to "crop" (is that a technical word - or casual english word?)

Reply:
That's an outcrop, yeah ;).

Cropping should be instantly recognizable by anyone who's ever done even the tiniest bit of image manipulation or photography.

Cropping allows you to crop something out.

And crop can be synonymous to cut in a number of contexts ;).